src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/InsertOperationTest.java

```diff
@@ -444,72 +341,60 @@ public void insertManyUnordered() throws Exception {
+                            "boolean": false,
+                            "nullval" : null,
+                            "array" : ["c", "d"],
+                            "sub_doc" : {"col": "lav"}
+                          }
+                          """;
 
       JsonNode jsonNode1 = objectMapper.readTree(document1);
       WritableShreddedDocument shredDocument1 = shredder.shred(jsonNode1);
 
       JsonNode jsonNode2 = objectMapper.readTree(document2);
       WritableShreddedDocument shredDocument2 = shredder.shred(jsonNode2);
 
-      String insertCql = INSERT_CQL.formatted(KEYSPACE_NAME, COLLECTION_NAME);
-      ValidatingStargateBridge.QueryAssert insertFirstAssert =
-          withQuery(
-                  insertCql,
-                  Values.of(CustomValueSerializers.getDocumentIdValue(shredDocument1.id())),
-                  Values.of(shredDocument1.docJson()),
-                  Values.of(CustomValueSerializers.getSetValue(shredDocument1.existKeys())),
-                  Values.of(CustomValueSerializers.getIntegerMapValues(shredDocument1.arraySize())),
-                  Values.of(
-                      CustomValueSerializers.getStringSetValue(shredDocument1.arrayContains())),
-                  Values.of(
-                      CustomValueSerializers.getBooleanMapValues(shredDocument1.queryBoolValues())),
-                  Values.of(
-                      CustomValueSerializers.getDoubleMapValues(
-                          shredDocument1.queryNumberValues())),
-                  Values.of(
-                      CustomValueSerializers.getStringMapValues(shredDocument1.queryTextValues())),
-                  Values.of(CustomValueSerializers.getSetValue(shredDocument1.queryNullValues())),
-                  Values.of(
-                      CustomValueSerializers.getTimestampMapValues(
-                          shredDocument1.queryTimestampValues())))
-              .withColumnSpec(
-                  List.of(
-                      QueryOuterClass.ColumnSpec.newBuilder()
-                          .setName("applied")
-                          .setType(TypeSpecs.BOOLEAN)
-                          .build()))
-              .withSerialConsistency(queriesConfig.serialConsistency())
-              .returningFailure(new RuntimeException("Ivan breaks the test."));
-
-      InsertOperation operation =
-          new InsertOperation(COMMAND_CONTEXT, List.of(shredDocument1, shredDocument2), true);
+      SimpleStatement insertStmt1 = nonVectorInsertStatement(shredDocument1);
+      SimpleStatement insertStmt2 = nonVectorInsertStatement(shredDocument2);
+
+      final AtomicInteger callCount = new AtomicInteger();
+      QueryExecutor queryExecutor = mock(QueryExecutor.class);
+
+      when(queryExecutor.executeWrite(eq(insertStmt1)))
+          .then(
+              invocation -> {
+                callCount.addAndGet(1);
+                return Uni.createFrom().failure(new RuntimeException("Ivan breaks the test."));
```

Comment:
I know these messages were from before. If possible can we change the message with personal name?

Reply:
Yeah I started thinking about this towards the end. I'll change these, as fun as they were as historical reminders. :)